### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by PrajaktaPurohit**

### CODE REVIEW

**Comment by mwrock** [comment]
:+1:

File: lita_config.rb.example [hunk 1]
```diff
@@ -1,5 +1,3 @@
-require "lita-slack"
```
Comment (ksubrama) [inline lita_config.rb.example]: So you actually need this bit if you are setting up slack integration.  Maybe comment it out saying "uncomment once you put in slack tokens below"

Comment (PrajaktaPurohit) [inline lita_config.rb.example]: will do

File: lita_config.rb.example [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
-
+#Uncomment the following lines once Slack tockens are added below
```
Comment (ksubrama) [inline lita_config.rb.example]: typo: tokens

File: lita_config.rb.example [hunk 3]
```diff
@@ -1,5 +1,5 @@
...
+#-require "lita-slack"
```
Comment (ksubrama) [inline lita_config.rb.example]: no - after #

**Comment by ksubrama** [comment]
Few minor nits but :+1: otherwise.

**Pull request merged**